lib/acts-as-messageable/scopes.rb

```diff
@@ -12,7 +12,11 @@ module ClassMethods
       def initialize_scopes
         scope :are_from,          lambda { |*args| where(:sent_messageable_id => args.first, :sent_messageable_type => args.first.class.name) }
         scope :are_to,            lambda { |*args| where(:received_messageable_id => args.first, :received_messageable_type => args.first.class.name) }
-        scope :search,            lambda { |*args|  where("body like :search_txt or topic like :search_txt",:search_txt => "%#{args.first}%")}
+        if defined? search
+          scope :aam_search,      lambda { |*args|  where("body like :search_txt or topic like :search_txt",:search_txt => "%#{args.first}%")}
+        else
+          scope :search,          lambda { |*args|  where("body like :search_txt or topic like :search_txt",:search_txt => "%#{args.first}%")}
```

Comment:
Line is too long. [142/80]<br>Use the new Ruby 1.9 hash syntax.<br>Space missing after comma.<br>Space missing inside }.